src/main/java/frc/robot/subsystems/Elevator/Elevator.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems.Elevator;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+
+public class Elevator extends SubsystemBase {
+    private final WPI_TalonFX motor = new WPI_TalonFX(Ports.Elevator.MOTOR);
+    private static Elevator INSTANCE = null;

```

Comment:
The order, IMO, should be:
- elevator instance
- blank line
- motor
- unit model